charcoal/Snakefile

```diff
@@ -309,14 +309,15 @@ rule contigs_sig:
             {params.moltype} {input} -o {output}
     """
 
+
+
 # run a search, query.x.database.
-rule prefetch_all:
+rule prefetch_all_matches_wc:
     input:
         query = stage1_dir + '/{filename}.sig',
         databases = config['gather_db']
     output:
         csv = stage1_dir + '/{filename}.matches.csv',
```

Comment:
trying to grok why this is `{filename}` and the other is `{g}`. Using a comment as a placeholder as i figure this out.

Reply:
Ya I don't understand why this is different than what appears in e.g. line 338

Reply:
oh, that's a red herring - sorry, I should have made this 'g' as well.

Reply:
(they are indeed the same)

Reply:
I'll fix this and ask for a re-review :)

Reply:
fixed in f4271dcfc34.

as a note, the addition of the `prefetch_all_matches` rule as a concrete rule beneath this one was the important bit.